ios/Sources/AboutFeature/AboutScreen.swift

```diff
@@ -1,16 +1,109 @@
+import ComposableArchitecture
 import SwiftUI
 import Styleguide
 
 public struct AboutScreen: View {
-    public init() {}
+
+    private let store: Store<AboutState, AboutAction>
+
+    public init(store: Store<AboutState, AboutAction>) {
+        self.store = store
+        let fontDescriptor = UIFont.preferredFont(forTextStyle: .subheadline).fontDescriptor.withSymbolicTraits(.traitBold)!
+        let font = UIFont(descriptor: fontDescriptor, size: 0)
+        UISegmentedControl.appearance().selectedSegmentTintColor = UIColor(AssetColor.secondary.color)
+        UISegmentedControl.appearance().backgroundColor = UIColor(AssetColor.Background.secondary.color)
+
+        UISegmentedControl.appearance().setTitleTextAttributes([
+            .font: font,
+            .foregroundColor: UIColor(AssetColor.Base.primary.color)
+        ], for: .normal)
+
+        UISegmentedControl.appearance().setTitleTextAttributes([
+            .font: font,
+            .foregroundColor: UIColor(AssetColor.Base.white.color),
+        ], for: .selected)
+    }
 
     public var body: some View {
-        Text(L10n.AboutScreen.title)
+        NavigationView {
+            VStack {
+                WithViewStore(store) { viewStore in
+                    Picker(
+                        "",
+                        selection:
+                            viewStore.binding(
+                                get: { $0.selectedType },
+                                send: { .selectedPicker($0) }
+                            )
+                    ) {
+                        ForEach(SelectedType.allCases, id: \.self) { (type) in
+                            Text(type.title).tag(type)
+                        }
+                    }
+                    .pickerStyle(SegmentedPickerStyle())
```

Comment:
Please consider using `toolbar(content:)` as #480 to keep a consistent design of the navigation bar. 🙏
Without a navigation bar, we cannot do something like `InlineTitleNavigationBarScrollView` (may need extra works for the segmented control) in #487.

We can set the segmented control's width by doing something like embedding `NavigationView` in a `GeometryReader`.

Reply:
@myihsan 
Thank you for reviewing!
I tried to revive NavigationBar and use Toolbar, but the height of NavigationBar deviated from the design.😢
Do you know any good way to adjust NavigationBar height?

## Screenshot (after change)
<img src="https://user-images.githubusercontent.com/18099465/121121085-e5eb9c00-c859-11eb-8b55-d648ac16c58a.png" width="300">

Reply:
@shun-sasano 
Thanks!

> Do you know any good way to adjust NavigationBar height?

I don't know any robust way to do that and even didn't notice the height is different from other screens. 
Let me talk to the designer.🙇‍♂️

Reply:
@shun-sasano  @myihsan my
I'm sorry. I made a mistake and modified the height of the Navigation Bar on the 'About' page.

Reply:
@myihsan @nobonobopurin
Thank you for your confirmation!😄